test/promql/tests/pipelinerun-cpumins.yaml

```diff
@@ -0,0 +1,127 @@
+# This is the main input for unit testing.
+# Only this file is passed as command line argument.
+
+evaluation_interval: 1m
+
+rule_files:
+  - ../../../prometheus/base/prometheus.rules.yaml
+
+tests:
+  # Test 1.
+  - interval: 1m
+    # Series data.
+    input_series:
+      - series: 'kube_pod_container_resource_limits{resource="memory", namespace="ns1", pod="ns1p1", container="ns1p1c1"}'
+        values: '2x10'
+      - series: 'kube_pod_container_resource_limits{resource="cpu", namespace="ns1", pod="ns1p1", container="ns1p1c1"}'
+        values: '0.5x10'
+      - series: 'kube_pod_labels{label_pipelines_appstudio_openshift_io_type="t1", namespace="ns1", pod="ns1p1"}'
+        values: '1x10'
+      - series: 'container_last_seen{namespace="ns1", pod="ns1p1", container="ns1p1c1"}'
+        values: '200+60x10'
+
+      - series: 'kube_pod_init_container_resource_limits{resource="memory", namespace="ns1", pod="ns1p1", container="ns1p1c2"}'
+        values: '2x10'
+      - series: 'kube_pod_init_container_resource_limits{resource="cpu", namespace="ns1", pod="ns1p1", container="ns1p1c2"}'
+        values: '0.5x10'
+      - series: 'container_last_seen{namespace="ns1", pod="ns1p1", container="ns1p1c2"}'
+        values: '100+60x10'
+
+      - series: 'kube_pod_container_resource_limits{resource="memory", namespace="ns2", pod="ns2p1", container="ns2p1c1"}'
+        values: '1 1 1 4 _ 5 _ 7 9 10 _'
```

Comment:
Since we are talking about predefined fixed limits, their value is not suppose to change, it is not a counter.
If my memory limit is `1000` it will remain `1000` for all the container's life.
So a better representation of resource limits would be:
```
- series: 'kube_pod_container_resource_limits{resource="memory", namespace="ns2", pod="ns2p1", container="ns2p1c1"}'
        values: '1000 1000 1000 1000 1000 1000 1000 1000 1000 1000'
```

Or 
```
- series: 'kube_pod_container_resource_limits{resource="memory", namespace="ns2", pod="ns2p1", container="ns2p1c1"}'
        values: '1000x10'
```

Reply:
> As Yfatch's PR covers all three of our efforts, we may be able to close this PR once it merges.

Sure, will close this once it merged